test/index.js

```diff
@@ -812,21 +813,19 @@ function cleanStack(stack, max) {
 function startLanguageServer(t, serverFilePath, cwd = '.') {
```

Comment:
The code that calls `startLanguageServer` seems to pass `cwd` as a file path: E.g., https://github.com/remcohaszing/unified-language-server/blob/tests-node-ipc/test/index.js#L567.
But here it is treated as a URL?

Reply:
I think you’re right, but I decided to revert this part. I’ll look into it in another PR.